plugins/doc_fragments/ldap.py

```diff
@@ -29,6 +29,20 @@ class ModuleDocFragment(object):
       - Set the path to PEM file with CA certs.
     type: path
     version_added: "6.5.0"
+  client_cert:
+    type: path
+    description:
+      - PEM formatted certificate chain file to be used for SSL client authentication.
+      - Required if I(client_key) is defined.
+    aliases: [certfile]
```

Comment:
Are the aliases really needed? I would avoid adding aliases if possible.

Reply:
They were in the code I copied. I do not see any reason to keep them.